src/Types.ts

```diff
@@ -432,7 +432,25 @@ export type FindAllProps = Pick<
     | "acceptCondition"
     | "ignoreUncontrolled"
     | "ignoreAccessibiliy"
-> & { container: HTMLElement; skipDefaultCheck?: boolean };
+> & {
+    container: HTMLElement;
+};
+
+export type FindSomeProps = Pick<
+    FindFocusableProps,
+    | "container"
+    | "currentElement"
+    | "isBackward"
+    | "includeProgrammaticallyFocusable"
+    | "acceptCondition"
+    | "ignoreUncontrolled"
+    | "ignoreAccessibiliy"
+> & {
+    container: HTMLElement;
+    onElement: FindElementCallback;
```

Comment:
Please add some documentation for `onElement` and `FindElementCallback` that the return boolean means that the search should stop